src/cmd/run.js

```diff
@@ -323,7 +328,16 @@ export class ExtensionRunner {
   }
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
-    const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefoxBinary: firefox});
+    const binaryArgs = [];
+    const {firefoxApp, firefox, startUrl} = this;
+    if (startUrl) {
+      const urls = Array.isArray(startUrl) ? startUrl : [startUrl];
+      for (const url of urls) {
+        binaryArgs.push('--url', `${url}`);
```

Comment:
nit: is `${url}` necessary? I think it can just be `url` (without back ticks) because Flow will guarantee that it is a string.